### DISCUSSION TITLE
Fuse Mount: cp command not working

### DISCUSSION THREAD

**Discussion by hunglsxx** [seaweedfs/seaweedfs#3323] [Q&A]
Hi guys, 
I using command: `cp sourcefile.png dessfile.png` but it not work with error:
<img width="794" alt="image" src="https://user-images.githubusercontent.com/50700597/179381087-bff4d6fe-d49a-4ade-af54-0460d9331e71.png">
I running mount via `weed mount -cacheCapacityMB=0 -concurrentWriters=128 -filer=127.0.0.1:8888 -dir=/swfs-mount -filer.path=/ -debug` and mount log here:
<img width="1218" alt="image" src="https://user-images.githubusercontent.com/50700597/179381139-5d3b4ebc-6cc9-4459-b5c9-12d80459dd7c.png">
Please help,
Thanks

**Comment by chrislusf** [comment]
can not reproduce.
